go.mod

```diff
@@ -4,6 +4,7 @@ go 1.21
 
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.7
+	github.com/PaulSonOfLars/gotgbot/v2 v2.0.0-rc.25
```

Comment:
If we need to switch to a new library, can we open another PR just with this change please?

Reply:
We could also look at https://github.com/gotd/td that looks well maintained.

Reply:
`github.com/gotd/td` is a library for writing telegram clients meant for personal accounts, not bot accounts. So I'm afraid we can't use that.
Do you have any concerns switching to `github.com/PaulSonOfLars/gotgbot/v2`? I've been using that library quite a bit and can say it's by far my favorite telegram bot lib.

Reply:
> Do you have any concerns switching to `github.com/PaulSonOfLars/gotgbot/v2`? I've been using that library quite a bit and can say it's by far my favorite telegram bot lib.

No I'm fine using this lib